site/mkdocs.yml

```diff
@@ -11,6 +11,7 @@ nav:
 - About: index.md
 - Get Started:
   - get-started/quickstart.md
+  - get-started/super-quickstart.md
```

Comment:
please move it after the Notebook samples. It contains the read scenario from the Notebook samples

Reply:
you are right it contains the read scenario,
but it also contains the quickstart and booting up Fybrik.
won't it be better to have it next to the quickstart as a "OneClickDemo"?
the description of the sequence of events also describes that there is a boot up of fybrik and read request by a user

Reply:
fixed as we discussed